tests/python/gpu/test_operator_gpu.py

```diff
@@ -251,9 +253,11 @@ def check_fft(shape):
         for exe in exe_list:
             exe.backward([out_grad])
         a = np.fft.ifft(out_grad_complex, n=None, axis=-1, norm=None)
-        assert_almost_equal(a.real, exe.grad_arrays[0].asnumpy()/shape[3],rtol=1e-3, atol=1e-6)
+        assert_almost_equal(a.real, exe.grad_arrays[0].asnumpy()/shape[3],rtol=1e-3, atol=1e-5)
 
-@with_seed(0)
+# @roshrini - removing fixed seed for this test. raised tolerance level.
+# Issue to track: https://github.com/apache/incubator-mxnet/issues/11730
```

Comment:
Feel free to remove those comments and simply link the issue in PR description.

Reply:
Removed comments. Issues are already linked in this PR description.